torchgeo/datasets/so2sat.py

```diff
@@ -24,19 +24,28 @@ class So2Sat(NonGeoDataset):
     acquired by the Sentinel-1 and Sentinel-2 remote sensing satellites, and a
     corresponding local climate zones (LCZ) label. The dataset is distributed over
     42 cities across different continents and cultural regions of the world, and comes
-    with a split into fully independent, non-overlapping training, validation,
-    and test sets.
+    with a variety of different splits.
 
-    This implementation focuses on the *2nd* version of the dataset as described in
-    the author's github repository https://github.com/zhu-xlab/So2Sat-LCZ42 and hosted
-    at https://mediatum.ub.tum.de/1483140. This version is identical to the first
-    version of the dataset but includes the test data. The splits are defined as
-    follows:
+    This implementation covers the *2nd* and *3rd* versions of the dataset as described
+    in the author's github repository: https://github.com/zhu-xlab/So2Sat-LCZ42.
+
+    The different versions are as follows:
+
+    Version 2: This version contains imagery from 52 cities and is split into
+        train/val/test as follows:
```

Comment:
```suggestion
    train/val/test as follows:
```
I think it's the sudden indentation that it doesn't like.